core/core_bind.cpp

```diff
@@ -466,8 +466,8 @@ bool OS::is_restart_on_exit_set() const {
 Vector<String> OS::get_restart_on_exit_arguments() const {
 	List<String> args = ::OS::get_singleton()->get_restart_on_exit_arguments();
 	Vector<String> args_vector;
-	for (List<String>::Element *E = args.front(); E; E = E->next()) {
-		args_vector.push_back(E->get());
+	for (const String &E : args) {
```

Comment:
We shouldn't reuse the same variable name here. Before, `E` was short for `List<String>::Element`, but now, it means `arg`.

```suggestion
	for (const String &arg : args) {
```

Reply:
Or whatever is applicable in that context; if nothing feels appropriate, something generic like `item` or `element` works just as well

Reply:
Thanks for the naming suggestion. I'm not very familiar with these code, so I prefer to use the original names to avoid issues like name conflicts. If you have a better name in mind, please feel free to suggest it.

Reply:
@YYF233333 For most of the time, you can just use the same name as the source, but in singular form.

Reply:
Resolve most of the naming issues except a few special cases that seem not pure naming issues, I think it’s best to address them in domain specific PRs.